cuBERT_topic_modelling/cuBERTopic.py

```diff
@@ -19,36 +27,118 @@ def __init__(self):
         self.original_topic_mapping = None
         self.new_topic_mapping = None
         self.final_topic_mapping = None
-
-    def create_embeddings(self, data):
+        
+    #TODO: find a way to not iterate through the series (slow on large inputs)
+    def fix_padding(self, sr):
+        # Remove all the padding from the end
+        trimmed_collections = list()
+        max_arr_length = -1
+        for i in sr:
```

Comment:
Here, iterating through a series is not ideal, need to find a way using compatible methods from cupy, cudf that work in parallel. (vectorize, apply_along_axis etc.)